Library/Homebrew/brew.sh

```diff
@@ -110,16 +110,18 @@ then
   # Don't change this from Mac OS X to match what macOS itself does in Safari on 10.12
   HOMEBREW_OS_USER_AGENT_VERSION="Mac OS X $HOMEBREW_MACOS_VERSION"
 
+  # The system Curl is too old for some modern HTTPS certificates on
+  # older macOS versions.
   printf -v HOMEBREW_MACOS_VERSION_NUMERIC "%02d%02d%02d" ${HOMEBREW_MACOS_VERSION//./ }
   if [[ "$HOMEBREW_MACOS_VERSION_NUMERIC" -lt "101000" ]]
   then
     HOMEBREW_SYSTEM_CURL_TOO_OLD="1"
```

Comment:
Is this still used anywhere now?

Reply:
Yes: `Library/Homebrew/extend/os/mac/development_tools.rb`